### DISCUSSION TITLE
Interact with Lizard through a web interface while RoSys is active

### DISCUSSION THREAD

**Discussion by rodja** [zauberzeug/rosys#135] [Ideas / Feature Requests]
Currently the Lizard `monitor.py` can only be used when the serial connection in RoSys is disabled. For deeper analysis it would be great to have the ability to interact with Lizard via commands while the robot is running an automation.

**Comment by falkoschindler** [comment]
Wishlist:

- [ ] serial communication page
- [ ] log view of incoming (and outgoing?) messages
- [ ] filter messages by search term (optionally case-invariant and/or regex?)
- [ ] input field with persistent command history (per user or global?)
